<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1108</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>491</width>
      <height>641</height>
     </rect>
    </property>
    <widget class="QWidget" name="controller" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>481</width>
       <height>251</height>
      </rect>
     </property>
     <widget class="QSlider" name="lowHSlider">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>391</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>179</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="highHSlider">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>110</y>
        <width>391</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>179</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="lowSSlider">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>140</y>
        <width>391</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="highSSlider">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>170</y>
        <width>391</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="lowVSlider">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>200</y>
        <width>391</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="highVSlider">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>230</y>
        <width>391</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lowH">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>lowH</string>
      </property>
     </widget>
     <widget class="QLabel" name="highV">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>highV</string>
      </property>
     </widget>
     <widget class="QLabel" name="lowV">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>lowV</string>
      </property>
     </widget>
     <widget class="QLabel" name="highS">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>highS</string>
      </property>
     </widget>
     <widget class="QLabel" name="lowS">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>lowS</string>
      </property>
     </widget>
     <widget class="QLabel" name="highH">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>highH</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget_6" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Paint RGB</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="paintRGB">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_7" native="true">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>10</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Brush RGB</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="brushRGB">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Multicolor: MTCOLOR</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>40</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Default mask color: blue</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="openWeb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>471</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Open webcam</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deletePaint">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete paint</string>
     </property>
    </widget>
    <widget class="QPushButton" name="togglePainter">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable</string>
     </property>
    </widget>
    <widget class="QLabel" name="userWeb">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>291</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="webMask">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>220</y>
       <width>291</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>611</width>
      <height>641</height>
     </rect>
    </property>
    <widget class="QLabel" name="paintedFrame">
     <property name="geometry">
      <rect>
       <x>38</x>
       <y>45</y>
       <width>531</width>
       <height>561</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>620</y>
       <width>571</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>RGB Format: INT INT INT. Cuvant cheie: MTCOLOR pentru afisare culori random ale desenului/dreptunghi.</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
